test_C4Game.py

```diff
@@ -0,0 +1,92 @@
+"""Unit tests for C4Game.py"""
+
+import unittest
+from C4Game import *
+
+class TestC4Game(unittest.TestCase):
+
+	def test_init(self):
+		"""Tests for C4Game.__init__ (therefore includes some testing of C4Game.createBoard and Player class"""
+		game = C4Game(2, 3, 2, 1, 0, "Maya", "Joey")
+		# Tests basic attributes
+		self.assertEqual(game.players, 2)
+		self.assertEqual(game.boardWidth, 3)
+		self.assertEqual(game.boardHeight, 2)
+		self.assertEqual(game.winLength, 1)
+		self.assertEqual(game.turn, 0)
+		self.assertEqual(len(game.playerArray), game.players)
+		# Tests that playerArray comes out as expected
+		player0Exp = Player(0, "Maya")
```

Comment:
The Exp on the end of this variable name seems confusing, it's just a Player Object, ya? Also I think you should test it from the game.playersArray instead of making a new object.

Reply:
ah I see, but the Color comes from the Game, so maybe something like this makes more sense.
testP0Name = "maya"
testP1Name = "joey"
player0Exp = Player(0, testP1Name)
...

But not a big deal :P

Reply:
Can you explain to me why naming the strings like that is better? :)
Exp stood for expected in my mind.  What did it evoke for you?